src/libs/blueprint/conduit_blueprint_o2mrelation.cpp

```diff
@@ -263,16 +263,30 @@ bool generate_offsets(conduit::Node &n,
     }
     else
     {
+        // int64_accessor sizes = n["sizes"].as_int64_accessor();
+        // n["offsets"].reset();
+        // n["offsets"].set(DataType::int64(sizes.number_of_elements()));
+        // int64_array offsets_vals = n["offsets"].value();
+
+        // for (index_t o = 0; o < sizes.number_of_elements(); ++o)
+        // {
+        //    offsets_vals[o] = (o > 0) ? offsets_vals[0-1] + sizes[o-1] : 0;
+        // }
```

Comment:
This fills `n["offsets"]` with nonsense low integers.  @cyrush , what am I doing wrong here?

Reply:
`offsets_vals[0-1]` instead intended offsets_vals[`o-1`]  ?

maybe we should use `o_idx` instead `o`

Reply:
AArgh.  Yes, that was the problem.  Using the better name helps.